app/app.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const app = express();
+const { getApiMsg, getTopics } = require("./controllers");
+const {
+  errorHandler500,
+  errorHandler400,
+} = require("./error-handling-controllers");
+
+app.use(express.json());
```

Comment:
app.use(express.json()) shouldn't be added yet, as there is not a use for it yet!